operator/docker/Dockerfile

```diff
@@ -1,11 +1,15 @@
-FROM amd64/debian:11.6 
+FROM debian:11.6
```

Comment:
out of interest, how does it know which one to pull as the base image?

Reply:
It's actually smart enough to figure it out based on the architecture that the image needs to run on. The multi-platform "images" are really an index file that point to a set of images - one for each supported platform. When you do `docker pull`, the docker client looks at the index and pulls the right platform-specific image.